osfoffline/utils/authentication.py

```diff
@@ -118,7 +123,7 @@ def log_in(self, user=None, username=None, password=None):
         user.logged_in = True
         try:
             save(session, user)
-        except SQLAlchemyError as e:
+        except SQLAlchemyError as e:  # noqa
```

Comment:
"noqa" tells flake (our syntax checker) to skip this line. Otherwise we get an error ("variable e is defined but never used").